latex/Specific Aims.tex

```diff
@@ -124,21 +124,25 @@
 % Aim 4: SAMPL -- Goal is learning from the above (a) where we are now, (b) why we fail/what we're missing, and (c) how to do better
 %I've tried to encapsulate that in these revised aims.
 
-{\bf \underline{Aim 1.} Generate new data for ``simple'' SAMPL blind challenges on physical property prediction}\\
+%{\bf \underline{Aim 1.} Generate new data for ``simple'' SAMPL blind challenges on physical property prediction}\\
+{\bf \underline{Aim 1.} Spur improvements in modeling protonation states, tautomers, and solvation via generation of new physical property datasets.}\\
```

Comment:
Could work! Just worried about putting the goal first, rather than the experimental summary.

Maybe "Collect new physical property datasets to assess accuracy and drive improvements in modeling of protonation states, tautomers, and forcefields"?